yt/frontends/ramses/fields.py

```diff
@@ -32,6 +32,8 @@
 rho_units = "code_density"
 vel_units = "code_velocity"
 pressure_units = "code_pressure"
+e_units = "code_mass * code_velocity**2 / code_time**2"
+am_units = "code_mass * code_velocity * code_length"
```

Comment:
Can you make these names a little more explicit? e.g. something like `ener_units` and `ang_mom_units`? I had to stop a few seconds to figure out what the current abbreviations mean while I was reading this and that's easy to avoid